apps/sessions-demo/src/components/Home/use-session.ts

```diff
@@ -160,6 +163,24 @@ const restoreSession = async (
     storedSession.sessionKey,
   );
 
+const fetchChainId = async (connection: Connection) => {
```

Comment:
shouldn't we do this is `@fogo/sessions-sdk` when constructing the message automatically?  Is there ever a use case for using a chain id for a chain that doesn't match the one you're connected to?

Reply:
Makes sense, when I move things to the react sdk I'll also set up a context provider which handles fetching this & the ALT